docs/run/error-mitigation-explanation.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b1267e87-e0b3-4934-b4b8-0265fe84adfd",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "# Error mitigation technique explanations\n",
```

Comment:
Hmm. I think we can come up with a better title 🤔 

How about just "Error mitigation techniques". Or "Error mitigation techniques available in Qiskit Runtime"? or "Qiskit Runtime error mitigation techniques"

Reply:
Went with "Error mitigation techniques"